package.json

```diff
@@ -26,7 +26,8 @@
     "@actions/core": "^1.2.6",
     "@actions/exec": "^1.0.3",
     "@actions/io": "^1.0.2",
-    "@actions/tool-cache": "^1.3.4"
+    "@actions/tool-cache": "^1.3.4",
+    "js-yaml": "^3.14.0"
```

Comment:
Adding such an heavy dependency for such an edge case is not acceptable.

Reply:
Yep, I've done it via `.split`/`.slice` at first, but decided that correctness is better (it adds < 100kb to the current 2.1mb). I'll redo it via `.split/.slice` if we'll decide to merge it.

Reply:
The `BUNDLER_VERSION` version doesn't need any parsing, so that seems better.